### PR TITLE
add logo to README

### PR DESCRIPTION
**Pull request by danielinteractive**

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -50,70 +51,86 @@ and custom cohort size, stopping and maximum increments rules:
...
+
+# Initialize the CRM model.
+model <- LogisticLogNormal(
+  mean = c(-0.85, 1),
+  cov =
+    matrix(c(1, -0.5, -0.5, 1),
+      nrow = 2
+    ),
+  refDose = 56
+)
```
Comment (wwojciech) [inline README.Rmd:65]: Formatting only:
```
model <- LogisticLogNormal(
  mean = c(-0.85, 1),
  cov = matrix(c(1, -0.5, -0.5, 1), nrow = 2),
  refDose = 56
)
```

**Review by wwojciech — APPROVED** [review]
Tx Daniel!

File: README.md [hunk 2]
```diff
@@ -23,84 +35,102 @@ You can install the stable release version of crmPack from CRAN with:
...
+with a 2-parameter logistic regression model, using a log normal prior
+distribution, and custom cohort size, stopping and maximum increments
+rules:
 
 ``` r
 library(crmPack)
-#> Warning: package 'crmPack' was built under R version 3.4.4
-#> Loading required package: ggplot2
-#> Warning: package 'ggplot2' was built under R version 3.4.4
+#> Lade nötiges Paket: ggplot2
```
Comment (wwojciech) [inline README.md:47]: @danielinteractive - should not be in english?

**Comment by danielinteractive** [comment]
thx @wwojciech that was helpful!

**Pull request merged**